packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -170,6 +175,22 @@ export const encryptAndSaveMetadata = async ({
     return providerInstance.setFileContent(fileName, encrypted);
 };
 
+export const exportMetadataToBip329File = () => (dispatch: Dispatch, getState: GetState) => {
+    const labelForSelectedAccount = selectLabelingDataForSelectedAccount(getState());
+    const labelsToExport = transformToBip329(labelForSelectedAccount as Slip15LikeInput);
+
+    try {
+        downloadAsJsonL(labelsToExport, 'suite_labels_export.jsonl');
```

Comment:
```suggestion
        downloadAsJsonL(labelsToExport, 'suite_labels_export.json');
```

I think its a typo right?

Reply:
It is actually `.jsonl` https://jsonlines.org/

No typo.